web/src/stream_edit_subscribers.js

```diff
@@ -224,6 +224,7 @@ function remove_subscriber({stream_id, target_user_id, $list_entry}) {
 
     function removal_success(data) {
         let message;
+        const user_name = people.get_by_user_id(target_user_id).full_name;
```

Comment:
This function can be moved to the if-block where it is used as this is only used at one place. Also, you can directly use `people.get_full_name` function here.